scripts/server.py

```diff
@@ -0,0 +1,38 @@
+#! /usr/bin/env python
+
+import os
+import logging
+from BaseHTTPServer import HTTPServer
+from SimpleHTTPServer import SimpleHTTPRequestHandler
+import main
+
+PORT = 54633
+PATH_TO_PROJECT = 'GeositeFramework/'
+
+logging.basicConfig(level=logging.INFO)
+
+class CustomRootHTTPServer(HTTPServer):
+    def __init__(self, base_path, *args, **kwargs):
+        HTTPServer.__init__(self, *args, **kwargs)
+        self.RequestHandlerClass.base_path = base_path
+
+class CustomRootHTTPRequestHandler(SimpleHTTPRequestHandler):
+    def translate_path(self, path):
+        words = filter(None, path.split('/'))
+        modified_path = self.base_path
+        for word in words:
+            modified_path = os.path.join(modified_path, word)
+        return modified_path
```

Comment:
I know this is from a previous PR, but can you add a comment to this function to describe it's purpose?